Lib/fontParts/base/font.py

```diff
@@ -1455,10 +2004,7 @@ def _interpolate(self, factor, minFont, maxFont,
             dstLayer.interpolate(factor, minLayer, maxLayer,
                                  round=round, suppressError=suppressError)
         if self.layerOrder:
-            if ufoLib.DEFAULT_LAYER_NAME in self.layerOrder:
```

Comment:
Very hesitant to change this, but see why it may be ok. @typemytype or @typesupply thoughts?

Reply:
Is there a glitch somewhere? Can't find this line in my original `font.py` file pulled from `master`.

Reply:
I see it: https://github.com/robotools/fontParts/blob/2bfa558107e9ab7c3e2ccfdec098a62dafae5784/Lib/fontParts/base/font.py#L1458

Reply:
For some reason, `git pull` didn't work, and I had to reclone the repo to see these lines remotely, which is why they were missing. It was never my intention to delete them, and so I've added them to reflect [2bfa558](https://github.com/robotools/fontParts/commit/2bfa558107e9ab7c3e2ccfdec098a62dafae5784).